docs.Jenkinsfile

```diff
@@ -0,0 +1,47 @@
+def get_agent() {
```

Comment:
Adding the `#!groovy` shebang should get us syntax highlighting on GitHub and IDEs